src/BlazorUI/Demo/Client/Shared/Shared/NavMenu.razor.scss

```diff
@@ -16,16 +16,17 @@
     flex-direction: column;
     width: rem($NavMenuWidth);
     max-width: rem($NavMenuWidth);
-    height: calc(100vh - rem($HeaderHeight));
     background-color: $bit-color-background-primary;
     top: calc(rem($HeaderHeight) + var(--bit-status-bar-height));
+    height: calc(100vh - (rem($HeaderHeight) + var(--bit-status-bar-height)));
 
     @include lt-lg {
         top: 0;
         inset: 0;
         height: 100vh;
         z-index: 13;
         position: fixed;
+        padding-top: var(--bit-status-bar-height);
```

Comment:
instead of padding, use this value for `top`

Reply:
The padding has better UI